terminus/builders/osm_city_builder.py

```diff
@@ -37,35 +37,30 @@ def get_city(self):
         # Get origin of the map (we use the center)
         tree = ET.parse(self.osm_map)
         bounds = tree.find('bounds').attrib
-        self.bounds = {'minlat': float(bounds['minlat']),
-                       'maxlat': float(bounds['maxlat']),
-                       'minlon': float(bounds['minlon']),
-                       'maxlon': float(bounds['maxlon'])}
+        self.bounds = {'origin': LatLon(float(bounds['minlat']), float(bounds['minlon'])),
+                       'corner': LatLon(float(bounds['maxlat']), float(bounds['maxlon']))}
 
-        self.map_origin = LatLon((self.bounds['minlat'] + self.bounds['maxlat']) / 2,
-                                 (self.bounds['minlon'] + self.bounds['maxlon']) / 2)
+        self.map_origin = self.bounds['origin'].middle_point(self.bounds['corner'])
 
         logger.debug("Map Origin: {0}".format(self.map_origin))
 
         # Calculate bounding box based on lat/lon
-        min_xy = self._translate_coords(self.bounds['minlat'],
-                                        self.bounds['minlon'])
-        max_xy = self._translate_coords(self.bounds['maxlat'],
-                                        self.bounds['maxlon'])
+        min_xy = self._translate_coords(self.bounds['origin'])
```

Comment:
We currently have this code:
```
        self.bounds['size'] = Point(2 * abs(min_xy.x),
                                    2 * abs(min_xy.y),
                                    0)
```

The idea would be to replace it for something along the lines of:
```
        self.bounds['size'] = 2 * abs(min_xy)
```
Since we already support the multiplication in the `Point` class we would only need to add the implementation for the `__abs__`  method (see https://docs.python.org/2/library/operator.html#operator.__abs__)